newIDE/app/src/MainFrame/EditorContainers/HomePage/LearnSection/index.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+import * as React from 'react';
+import { type HomeTab } from '../HomePageMenu';
+import { type TutorialCategory } from '../../../../Utils/GDevelopServices/Tutorial';
+import MainPage from './MainPage';
+import TutorialsCategoryPage from './TutorialsCategoryPage';
+import { Trans } from '@lingui/macro';
+import { TutorialContext } from '../../../../Tutorial/TutorialContext';
+import PlaceholderError from '../../../../UI/PlaceholderError';
+import PlaceholderLoader from '../../../../UI/PlaceholderLoader';
+
+export const TUTORIAL_CATEGORY_TEXTS = {
+  'full-game': {
+    title: <Trans>Entire games</Trans>,
+    description: <Trans>Make complete games step by step</Trans>,
+  },
+  'game-mechanic': {
+    title: <Trans>Specific game mechanics</Trans>,
+    description: (
+      <Trans>
+        Find how to implement the most common game mechanics and more!
```

Comment:
Nitpick: there is an exclamation mark here while the other descriptions do not have any punctuation